12_upgrade/00_upgrade.txt

```diff
@@ -14,8 +14,18 @@ occur during the upgrade process.
 
 Migration from 3.1 or 3.2 are supported, so you can upgrade directly to 4.0.
 
-If you migrate from a 3.1 release older than 3.1.11, you have to upgrade
-you server before your agents (as 4.0 agents are not compatible with old servers).
+[WARNING]
+
+====
+
+As we changed the communication protocol between agent end server in 4.0,
```

Comment:
Could you add a note to say the change remains compatible, to make it less scary.